clouddriver-kubernetes/src/integration/java/com/netflix/spinnaker/clouddriver/kubernetes/it/DeployManifestIT.java

```diff
@@ -1108,36 +1108,6 @@ public void shouldDeployRedBlackMultidoc() throws IOException, InterruptedExcept
     KubeTestUtils.disableManifest(baseUrl(), body, account1Ns, "replicaSet " + appName + "-v000");
 
     // ------------------------- then --------------------------
-    String port =
```

Comment:
I'm probably missing something...is there something different to do with kind to get the same behavior? Or, how does this "just work" now?

Reply:
Unfortunately you cannot use services of type `NodePort` in kind and expect them to be reachable from `localhost`, (at least in non-linux hosts), because it's running inside a virtual docker network. I opted for removing these checks that query the exposed service and only rely on reading the pods labels to assert that only one pod has the selector label.

Reply:
OK.  Maybe this is even a better test this way?  Like, if it fails then maybe there's some other place to wait for feedback?